CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ fixes via patches with patch version bumps.
 
 ## Unreleased
 
-Nothing yet.
+* Handle error case in Uncompact
```

Comment:
can you update in line with https://keepachangelog.com/en/1.0.0/ format?  
```
### Changed

* [breaking] `Uncompat` now returns `([]H3Index, error)` instead of `[]H3Index` to accommodate error scenario from C API.

### Fixed

* panic when using `Uncompact` with invalid resolutions.
```